Tests/AnimationResultSpec.swift

```diff
@@ -56,6 +56,24 @@ final class AnimationResultSpec: QuickSpec {
                 }
             }
 
+            it("motion animation along path should change both layer's X and Y positions") {
+                let path = UIBezierPath()
+                path.move(to: layer.position)
+                path.addLine(to: CGPoint(x: 45, y: 50))
+                path.addLine(to: CGPoint(x: 65, y: 63))
+                layer.lotus.runAnimation {
+                    $0.motion.along(path).during(0.01)
+                }
+
+                waitUntil(timeout: 0.5) { done in
```

Comment:
Looks weird to me. Why asyncAfter is not enough for the tests? I think we should find another way to test animations